### DISCUSSION TITLE
Overview doesn't works with footnote.

### DISCUSSION THREAD

**Discussion by tanreeve** [ahrm/sioyek#624] [Q&A] [answered]
Helllo, 
I tried to right click the footnote number, but it doesn't open the overview window. 
Does this app not support footnote overview?

my footnote like this: 

<img src="https://user-images.githubusercontent.com/114658849/216959356-ae148b8b-6a7e-4c94-bec0-edfb51a7b24f.png" width="600" height="300">

This example pdf file :

[test.pdf](https://github.com/ahrm/sioyek/files/10662496/test.pdf)

Thank you.

**Comment by ahrm** [comment, marked as answer]
We don't support footnotes for smartjumps (it works if it is linked in the PDF though).

**Reply by tanreeve** [reply]
Ok thank you for the answer.
